packages/dcos-ui-service/build

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+cp -r "/pkg/src/dcos-ui-service" "$PKG_PATH"/usr/
```

Comment:
Looks like we have to expand this, cluster started with no knowledge about our service

```suggestion
cp -r "/pkg/src/dcos-ui-service" "$PKG_PATH"/usr/

# Create the service file
service="$PKG_PATH/dcos.target.wants_master/dcos-ui-service.service"
mkdir -p "$(dirname "$service")"
cp /pkg/extra/dcos-ui-service.service  "$service"
```